Foundation/URLCache.swift

```diff
@@ -13,6 +13,13 @@ import SwiftFoundation
 import Foundation
 #endif
 
+internal extension NSLock {
+    func performLocked<T>(_ block: () throws -> T) rethrows -> T {
```

Comment:
`.synchronized` is an internal method in Foundation, and this code is in FoundationNetworking. — @karthikkeyan